packages/spatial/src/input/systems/ClientInputSystem.tsx

```diff
@@ -85,7 +85,7 @@ const DRAGGING_THRESHOLD = 0.001
 const ROTATING_THRESHOLD = 1.5 * (PI / 180)
 
 /** anti-garbage variable!! value not to be used unless you set values just before use*/
-const pointerPositionVector3 = new Vector3()
+const g_pointerPositionVector3 = new Vector3()
```

Comment:
we do not use this format for variables. please keep the naming conventions the same.

Reply:
kk will change them.
What could I do to clarify that these variables are access from the top of the file, and not sent into the functions or created by them?

Reply:
Fixed with 25c6b83